RELEASE-NOTES.rst

```diff
@@ -19,7 +19,7 @@ What's new
 Installation
 ------------
 
-   $ pip install invenio-db==1.0.0
+   $ pip install invenio-db==1.0.0a7
```

Comment:
For previous releases we haven't added the alpha part to the version number like we don't update the What's new section etc.